docker/wc-admin-php-test-suite/Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM alpine
+FROM alpine:3.13
```

Comment:
php 8 was not supported in alpine 3.12 and below: https://pkgs.alpinelinux.org/packages?name=php8&branch=v3.13

Reply:
I'm not sure we should pin the version, since then we can miss out on future security updates.

The way I understand it is the` image` version bump in `docker-compose.yml` should trigger the latest alpine image to be downloaded on the next use.

Reply:
Thanks, that indeed was fine, I removed it.